zebra-network/src/address_book.rs

```diff
@@ -419,18 +442,18 @@ impl AddressBook {
         // every request, use the trace-level logs, or the metrics exporter.
         if let Some(last_address_log) = self.last_address_log {
             // Avoid duplicate address logs
-            if Instant::now().duration_since(last_address_log).as_secs() < 60 {
+            if now.saturating_duration_since(last_address_log).as_secs() < 60 {
```

Comment:
This line panics without this change, but it shouldn't, because we've only replaced 3 time calls with the same time.